.build/unit-testing/run

```diff
@@ -1,36 +1,22 @@
 #!/bin/bash
 
-set -ux
+set -u
 
 function main() {
-  runTests
+  .build/setup-database
+  ./gradlew --quiet test jacocoTestReport --exclude-task :smoke-testing:test
```

Comment:
@DanVanAtta Could you explain why you decided to add the --quiet flag here?
When a test fails this flag causes it to not show the test that actually failed, which makes issues really hard to debug that don't happen locally